TimePicker/TimePicker.js

```diff
@@ -247,7 +247,7 @@ const TimePicker = Pure(
  * @function
  * @memberof sandstone/TimePicker
  * @param {Date} time `Date` to convert
- * @returns {String?} Converted date or `null` if `date` is invalid
+ * @returns {String|Null} Converted date or `null` if `date` is invalid
```

Comment:
How will this change affect the resulting doc? If you change the doc, please provide a link to the new document.